app/src/main/java/com/example/livedata/LoginViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.livedata
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import android.view.View
+
+class LoginViewModel : ViewModel() {
+    private val mutableEmailAddress = MutableLiveData<String>()
+    private val mutablePassword = MutableLiveData<String>()
+    val emailAddress: LiveData<String> = mutableEmailAddress
+    val password: LiveData<String> = mutablePassword
+
+    private var userMutableLiveData: MutableLiveData<User> = MutableLiveData()
+
+    val user: MutableLiveData<User>
```

Comment:
Is there a reason this isn't in the same format as the other live data fields? i.e.
```
private val mutableUser = MutableLiveData<User>()
val user = mutableUser
```